### PR TITLE
Add Works and About sections

### PR DESCRIPTION
**Pull request by lu-jim**
Second PR for this project, per the Microverse specifications.
## Changes
- Added Works section with CSS Grid
- Added About section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,27 +22,160 @@
...
+                <img src="./resources/MediumVector.svg" alt="Medium logo"></a></li>
+            <li><a href="https://github.com/lu-jim" target="_blank" rel="noopener noreferrer">
+                <img src="./resources/GithubVector.svg" alt="Github logo"></a></li>
+            <li><a href="https://angel.co" target="_blank" rel="noopener noreferrer">
+                <img src="./resources/AngelListVector.svg" alt="AngelList logo"></a></li>
+          </ul>
+        </div>
+      </section>
+    </header>
+    <div class="works">
```
Comment (Alexoid1) [inline index.html]: - [x] Kindly replace the `div` tag with section `tag`.

File: index.html [hunk 2]
```diff
@@ -20,27 +22,160 @@
...
+      <article class="card-1">
```
Comment (Alexoid1) [inline index.html:51]: - [x] Add a title "My Recent Works" with h2 tag for the projects section.

File: index.html [hunk 3]
```diff
@@ -20,27 +22,160 @@
...
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+
+      <article class="card-2">
```
Comment (Alexoid1) [inline index.html:73]: - [x] Kindly fix the styling for the projects section for mobile version.
Captura de pantalla_2021-09-19_00-04-28
Captura de pantalla_2021-09-19_00-05-20

-  kindly read this article in order to learn how to use Chrome dev tools to [Simulate mobile devices with Device Mode](https://developer.chrome.com/docs/devtools/device-mode/).

File: index.html [hunk 4]
```diff
@@ -20,27 +22,160 @@
...
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
```
Comment (Alexoid1) [inline index.html:90]: - Kindly replace `div` with `ul `and `li` tags to list the technologies.

File: index.html [hunk 5]
```diff
@@ -20,27 +22,160 @@
...
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
```
Comment (Alexoid1) [inline index.html]: - [x] Kindly replace `div` with `ul `and `li` tags to list the technologies.

File: index.html [hunk 6]
```diff
@@ -20,27 +22,160 @@
...
```
Comment (Alexoid1) [inline index.html:134]: - [x] Kindly replace `div` with `ul `and `li` tags to list the technologies.

File: index.html [hunk 7]
```diff
@@ -20,27 +22,160 @@
...
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+    </div>
+    <div class="about">
       <h1 class="title">
```
Comment (Alexoid1) [inline index.html]: - [x] Replace `h1` with a `h2` tag. The `h1` tag is just for the title of the website.

File: index.html [hunk 8]
```diff
@@ -20,27 +22,160 @@
...
+        contact me.</p>
       <h2 class="connect"> <a href="#">Let's connect</a></h2>
       <ul class="socials">
         <li><a href="https://twitter.com/lujimhe" target="_blank" rel="noopener noreferrer">
-          <img src="./resources/TwitterVector.svg" alt="Twitter logo" ></a></li>
+            <img src="./resources/TwitterVector.svg" alt="Twitter logo"></a></li>
         <li><a href="https://www.linkedin.com/in/lujim/" target="_blank" rel="noopener noreferrer">
-          <img src="./resources/LinkedinVector.svg" alt="Linkedin logo"></a></li>
+            <img src="./resources/LinkedinVector.svg" alt="Linkedin logo"></a></li>
         <li><a href="https://medium.com/@lu.jim" target="_blank" rel="noopener noreferrer">
```
Comment (Alexoid1) [inline index.html]: - [x] Kindly fix the styling in the about section for mobile version.
Captura de pantalla_2021-09-19_00-28-00
Captura de pantalla_2021-09-19_00-29-19

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
# Hi @lu-jim,

Impressive project!
You are doing a great job so far, and the project is looking good.

## Required Changes :recycle:
 For the project to get approved, also implement the changes mentioned below.

## Optional suggestions
 - Kindly read this article in order to learn how to use Chrome dev tools to [Simulate mobile devices with Device Mode](https://developer.chrome.com/docs/devtools/device-mode/).

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (lu-jim) [reply index.html:51]: Hello! Thanks for the review, I can certainly add this, but it is not part of the template I chose (Template 1) so I'm unsure where to place it.
image

File: index.html (hunk 5 above)
Comment (lu-jim) [reply index.html]: Hello! My code already had these technologies as ul and li. Am I missing where should I change this?
```
<ul class="skill-list">
        <li class="list-title">Languages <img src="./resources/downarrow.svg" alt="Down Arrow"></li>
      </ul>
        <ul class="sublist">
          <li class="subitem"> <img src="./resources/jsicon.svg" alt="JavaScript" class="skill-logo"> JavaScript</li>
          <li class="subitem"> <img src="./resources/html5.svg" alt="HTML" class="skill-logo"> HTML</li>
          <li class="subitem"> <img src="./resources/css.svg" alt="CSS" class="skill-logo"> CSS</li>
        </ul>
      <ul class="skill-list">
        <li class="list-title">Frameworks <img src="./resources/rightarrow.svg" alt="Right Arrow"></li>
        <li class="list-title">Skills <img src="./resources/rightarrow.svg" alt="Down Right"></li>
      </ul>
```

File: index.html (hunk 2 above)
Comment (Alexoid1) [reply index.html:51]: - Okay just leave it like that and please continue with the other requested changes.

Comment (lu-jim) [reply index.html:51]: Done!

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @lu-jim, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (Alexoid1) [reply index.html]: - HTML file line 112

File: index.html [hunk 9]
```diff
@@ -20,27 +22,160 @@
...
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
```
Comment (mke2111) [inline index.html:138]: - [x] As preciously requested, implement this feature for technology stacks listed. Do this for all your projects
> Kindly replace `div` with `ul` and `li` tags to list the technologies.

```
        <ul>
          <li>html</li>
          <li>Javascript</li>
          <li>CSS</li>
        </ul>
```

File: index.html (hunk 5 above)
Comment (lu-jim) [reply index.html]: Thanks! I fixed it now, apologies for missing it earlier.

File: index.html (hunk 9 above)
Comment (lu-jim) [reply index.html:138]: Just saw where I missed this in the previous PR. Fixed now.

File: index.html [hunk 10]
```diff
@@ -20,27 +22,160 @@
...
+    </section>
+    <div class="about">
+      <h2 class="title">
+        About Myself
+      </h2>
+      <p class="descr">Hello! I’m a software developer! I can help you build a product, feature or website. Look through
+        some
+        of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+        contact me.</p>
+      <h3 class="connect"> <a href="#">Let's connect</a></h3>
```
Comment (mke2111) [inline index.html:150]: - [x] As previously requested, the styling for the about section is a little offf. Compare these two images, especially the padding.
YOUR PAGE Vs ORIGINAL TEMPLATE
<div class="flex">


</div>

**Pull request merged**